extensions/adobe/experience/decisioning/profile-constraint-details.schema.json

```diff
@@ -35,8 +35,8 @@
           "meta:enum": {
             "none": "",
             "eligibilityRule": "The profile constraint is expressed as a single rule that must evaluate to true before the constrained action is allowed",
-            "Any Segments": "The profile constraint is expressed as one or more segments and the profile must be a member of at least one of them before the constrained action is allowed",
-            "All Segments": "The profile constraint is expressed as one or more segments and the profile must be a member of all of them before the constrained action is allowed",
+            "anySegments": "The profile constraint is expressed as one or more segments and the profile must be a member of at least one of them before the constrained action is allowed",
+            "allSegments": "The profile constraint is expressed as one or more segments and the profile must be a member of all of them before the constrained action is allowed",
             "rules": "The profile constraint is expressed as a number of different rules, e.g. eligibility, applicability, suitability, which all must evaluate to true before the constrained action is allowed"
```

Comment:
These meta:enums seem odd- @mjasrotia / @prabhum2 - any idea where they came from so we can track down if this is the desired label on the meta:enum?

Reply:
still need confirmation from Manik or Prabhakar on this one